### PR TITLE
PROGRAMMES-6154 Speed up test on jenkins

### PR DESCRIPTION
**Pull request by tatitati**
WITH_TEST_COVERAGE is the new boolean parameter setup in jenkins. This allows to the user to choose whether running tests with coverage or not

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: script/ci-docker [hunk 1]
```diff
@@ -33,7 +33,11 @@ php --version
 yarn gulp-ci
 
 # Run PHPUnit
-bin/phpunit || RESULT=1
+if [ $WITH_TEST_COVERAGE == true ]; then
+    bin/phpunit || RESULT=1
+else
+    bin/phpunit --no-coverage || RESULT=1
+fi
```
Comment (tatitati) [inline script/ci-docker]: Should I add some kind of ```echo 'running with code coverage'``` and ```echo 'running without code coverage'``` to make clearer the output and have some kind of "reminder" about how was ran a build?

Do we want to add also something like ```time bin/phpunit ....``` to check better how long they take

File: script/ci-docker [hunk 2]
```diff
@@ -33,7 +33,11 @@ php --version
 yarn gulp-ci
 
 # Run PHPUnit
-bin/phpunit || RESULT=1
+if [ $WITH_TEST_COVERAGE == true ]; then
```
Comment (BjornTwachtmann) [inline script/ci-docker]: This will test for the string true, rather than a boolean value. Is that what you want?

File: script/ci-docker (hunk 1 above)
Comment (BjornTwachtmann) [reply script/ci-docker]: Yeah, sounds useful

File: script/ci-docker [hunk 3]
```diff
@@ -33,7 +33,13 @@ php --version
 yarn gulp-ci
 
 # Run PHPUnit
-bin/phpunit || RESULT=1
+if $WITH_TEST_COVERAGE ; then
```
Comment (BjornTwachtmann) [inline script/ci-docker]: I think this should probably read 
```if [[ ! -z "$WITH_TEST_COVERAGE" ]]```
That tests for any non-empty string

Comment (tatitati) [reply script/ci-docker]: I tested it. I guess that you were thinking that the value is set only when the checkbox is checked. To make sure of this I tested your idea, but is matching always the same condition ($WITH_TEST_COVERAGE has always a value, when checked or unchecked).

Comment (BjornTwachtmann) [reply script/ci-docker]: Fair enough. If it works.

**Pull request merged**